test/src/Provider/AbstractProviderTest.php

```diff
@@ -507,4 +477,50 @@ public function testInvalidAccessTokenMethod()
         $provider->setAccessTokenMethod('PUT');
         $provider->getAccessToken('authorization_code', ['code' => 'mock_authorization_code']);
     }
+
+    public function getPrivateMethod($class, $name)
+    {
+        $class = new \ReflectionClass($class);
+        $method = $class->getMethod($name);
+        $this->assertFalse($method->isPublic());
+        $method->setAccessible(true);
+        return $method;
+    }
+
+    private function _testParse($body, $type, $expected = null)
```

Comment:
@shadowhand is there a nicer way to do this? Makes for nice, small parse test cases though. :)